.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+**[Mailing List](https://groups.google.com/forum/#!forum/liftweb) thread:** REPLACE THIS WITH ML LINK
+
+**Example project link:** LINK TO EXAMPLE PROJECT IF APPLICABLE
```

Comment:
Yeah. Hm. Let's table that for now. I'll just link to the assembla site. Fixing the diaspora of documentation is a larger problem that the community needs to make some decisions about before we move forward.